pkgs/games/armagetronad/default.nix

```diff
@@ -1,42 +1,77 @@
-{ lib, stdenv, fetchurl
-, pkg-config, SDL, libxml2, SDL_image, libjpeg, libpng, libGLU, libGL, zlib
-, dedicatedServer ? false }:
+{ lib
+, stdenv
+, fetchurl
+, pkg-config
+, SDL
+, libxml2
+, SDL_image
+, libjpeg
+, libpng
+, libGLU
+, libGL
+, zlib
+, dedicatedServer ? false
+}:
 
 let
   versionMajor = "0.2.9";
-  versionMinor = "1.0";
+  versionMinor = "1.1";
   version = "${versionMajor}.${versionMinor}";
+  drv = dedicatedServer:
+    stdenv.mkDerivation rec {
+      pname = if dedicatedServer then "armagetronad-dedicated" else "armagetronad";
+      inherit version;
+      src = fetchurl {
+        url = "https://launchpad.net/armagetronad/${versionMajor}/${version}/+download/armagetronad-${version}.tbz";
+        hash = "sha256-9hfecA7PHdEfdeKTLudHlnKREjUtXKuqK8R5rdff/TI=";
+      };
+
+      enableParallelBuilding = true;
+
+      configureFlags = [
+        "--enable-automakedefaults"
+        "--disable-memmanager"
+        "--disable-useradd"
+        "--disable-initscripts"
+        "--disable-etc"
+        "--disable-uninstall"
+        "--disable-sysinstall"
+      ] ++ lib.optional dedicatedServer "--enable-dedicated";
+
+      nativeBuildInputs = [ pkg-config ];
+
+      buildInputs = [ libxml2 zlib ]
+        ++ lib.optionals (!dedicatedServer) [ SDL SDL_image libxml2 libjpeg libpng libGLU libGL ];
+
+      # Just use make all.
+      buildPhase = "make -j";
+
+      # Don't parallelize this to avoid a race.
+      installPhase = "make -j1 install";
```

Comment:
Is there a nicer way to ask stdenv to enable parallel building but not for make install?

Reply:
```nix
  enableParallelBuilding = true;
  enableParallelInstalling = false;
```

Reply:
Too easy. Thanks!